joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/ClassTests.scala

```diff
@@ -10,7 +10,7 @@ import io.joern.rubysrc2cpg.passes.Defines.{Initialize, Main, TypeDeclBody}
 import io.joern.rubysrc2cpg.passes.GlobalTypes
 import io.shiftleft.codepropertygraph.generated.NodeTypes
 
-class ClassTests extends RubyCode2CpgFixture {
+class ClassTests extends RubyCode2CpgFixture(useJsonAst = false) {
```

Comment:
If this is meant to be set to `false`, you can just remove it since it defaults to `false`

Reply:
Kept it in for now as it has partial passing tests and is currently what I'm working on, this PR is mostly a progress push to avoid merge issues later on regarding changes to `RubySrc2Cpg.scala`